### DISCUSSION TITLE
iPXE booting to Fog server

### DISCUSSION THREAD

**Discussion by mazaffar1** [ipxe/ipxe#611] [Q&A]
Hello

I am trying to pxe boot to the fig server to install a W10 image.
The HP laptop is using iPXE and does not get a response from the DHCP server.  I am wondering if iPXE is not compatible with the Fog server.  Can I turn off iPXE and use PXE instead?

**Comment by NiKiZe** [comment]
FOG uses iPXE, but ask them for alternatives if you want.

Otherwise if you post details of your issues, maybe someone can help you solve them.

**Comment by mazaffar1** [comment]
Don't understand what you mean.

On Fri, 11 Mar 2022, 09:36 Christian Nilsson, ***@***.***>
wrote:

> FOG uses iPXE, but ask them for alternatives if you want.
>
> Otherwise if you post details of your issues, maybe someone can help you
> solve them.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ipxe/ipxe/discussions/611#discussioncomment-2337950>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYFR4USDTT4TWQWCY7IR34DU7MHYXANCNFSM5QPAANKQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 


Confidentiality Notice

 

This email is private and confidential and may 
be legally privileged. If you have received this email in error, please 
send it back to us, and immediately and permanently delete it. Do not use, 
copy or disclose the information contained in this email or in any 
attachment.

 

Co-op Academies Trust uses security software which 
routinely monitors and sometimes blocks emails. Further monitoring may be 
conducted for general business or security purposes, but only where 
lawfully permitted. Co-op Academies Trust cannot accept liability for any 
damage which you may sustain as a result of software viruses or 
interception or interruption of email, so please carry out you own virus 
checks before opening an attachment. 

 

Co-op Academies Trust is a 
working name of the The Co-operative Academies Trust which is a 
multi-academy trust.

**Reply by NiKiZe** [reply]
You are asking iPXE for alternatives to iPXE to use with FOG, you should ask FOG that question. Not iPXE.

If you are asking for help with iPXE then you need to explain what you are seeing. We don't know anything about your device, or your setup.

**Comment by stappersg** [comment]
On Fri, Mar 11, 2022 at 01:39:12AM -0800, mazaffar1 wrote:
>  Don't understand what you mean.

It is explained at http://www.catb.org/~esr/faqs/smart-questions.html
Please read that webpage and use your new knowledge.

**Comment by mazaffar1** [comment]
thank you

On Fri, 11 Mar 2022 at 14:56, Geert Stappers ***@***.***>
wrote:

> On Fri, Mar 11, 2022 at 01:39:12AM -0800, mazaffar1 wrote:
> > Don't understand what you mean.
>
> It is explained at http://www.catb.org/~esr/faqs/smart-questions.html
> Please read that webpage and use your new knowledge.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ipxe/ipxe/discussions/611#discussioncomment-2339778>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYFR4UQU2MIFPMRCZ2IIAJTU7NNIJANCNFSM5QPAANKQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 


Confidentiality Notice

 

This email is private and confidential and may 
be legally privileged. If you have received this email in error, please 
send it back to us, and immediately and permanently delete it. Do not use, 
copy or disclose the information contained in this email or in any 
attachment.

 

Co-op Academies Trust uses security software which 
routinely monitors and sometimes blocks emails. Further monitoring may be 
conducted for general business or security purposes, but only where 
lawfully permitted. Co-op Academies Trust cannot accept liability for any 
damage which you may sustain as a result of software viruses or 
interception or interruption of email, so please carry out you own virus 
checks before opening an attachment. 

 

Co-op Academies Trust is a 
working name of the The Co-operative Academies Trust which is a 
multi-academy trust.

**Comment by mazaffar1** [comment]
we resolved it by plugging in a usb adapter that has an ethernet port to
the laptop and pxe booted.

On Fri, 11 Mar 2022 at 11:51, Christian Nilsson ***@***.***>
wrote:

> You are asking iPXE for alternatives to iPXE to use with FOG, you should
> ask FOG that question. Not iPXE.
>
> If you are asking for help with iPXE then you need to explain what you are
> seeing. We don't know anything about your device, or your setup.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ipxe/ipxe/discussions/611#discussioncomment-2338729>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYFR4UTMDMMGV7LCYL37SGLU7MXSTANCNFSM5QPAANKQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 


Confidentiality Notice

 

This email is private and confidential and may 
be legally privileged. If you have received this email in error, please 
send it back to us, and immediately and permanently delete it. Do not use, 
copy or disclose the information contained in this email or in any 
attachment.

 

Co-op Academies Trust uses security software which 
routinely monitors and sometimes blocks emails. Further monitoring may be 
conducted for general business or security purposes, but only where 
lawfully permitted. Co-op Academies Trust cannot accept liability for any 
damage which you may sustain as a result of software viruses or 
interception or interruption of email, so please carry out you own virus 
checks before opening an attachment. 

 

Co-op Academies Trust is a 
working name of the The Co-operative Academies Trust which is a 
multi-academy trust.
